### PR TITLE
Fixed audit warnings for npm and copy of homepage

### PR DESCRIPTION
**Pull request by divya-mohan0209**
This PR does the following:
- Clean the homepage copy & add a button so that it looks as below
image

- Amend the audit warnings for npm due to version. This is reflected in package.json.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default function Home() {
     <>
       <Head>
         <title>SLE Base Container Images</title>
-        <meta name='description' content='Minimal SUSE Linux Enterprise Server 15-based images that you can use to develop, deploy, and share applications' />
+        <meta name='description' content='Secure software innovation' />
```
Comment (dirkmueller) [inline pages/index.tsx:12]: Why this change? Previous text appeared more matching for me?

Comment (divya-mohan0209) [reply pages/index.tsx:12]: From a copy perspective, the original text was very verbose. We need a maximum of three/four words for this section since this is like a tagline of-sorts

**Review by mattfarina — APPROVED** [review]
